libraries/AP_NavEKF3/AP_NavEKF3_Control.cpp

```diff
@@ -527,6 +537,7 @@ bool NavEKF3_core::use_compass(void) const
 // are we using a yaw source other than the magnetomer?
 bool NavEKF3_core::using_external_yaw(void) const
 {
+#if EK3_FEATURE_EXTERNAL_NAV
```

Comment:
I think this might not be correct.  My understanding is that "using_external_yaw()" should return true if GPS is being used for yaw (i.e. GPS-for-yaw or GSF) this the change in this PR means it will always return false which will cause pre-arm checks to fail.